test/rekt/features/broker_config.go

```diff
@@ -53,11 +53,11 @@ func BrokerWithCustomReplicationFactorAndNumPartitions(env environment.Environme
 	))
 	f.Setup("Broker ready", broker.IsReady(brokerName))
 
-	topic := kafka.BrokerTopic(brokerreconciler.TopicPrefix, &eventing.Broker{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      brokerName,
-			Namespace: env.Namespace(),
-		},
+	topic, err := apisconfig.DefaultFeaturesConfig().ExecuteBrokersTopicTemplate(metav1.ObjectMeta{Name: brokerName, Namespace: env.Namespace()})
+	f.Assert("No error creating broker topic", func(ctx context.Context, t feature.T) {
+		if err != nil {
+			t.Fatal(err)
+		}
 	})
```

Comment:
Same question here, what are we trying to test here?

Reply:
Same answer as above